static/js/containers/FinancialAidCalculator.js

```diff
@@ -56,7 +55,7 @@ const currencySelect = (update, current) => (
 
 const salaryUpdate = R.curry((update, current, e) => {
   let newEdit = R.clone(current);
-  newEdit.income = sanitizeNumberString(20, e.target.value);
+  newEdit.income = e.target.value
```

Comment:
Oh, that makes sense. How about we add the error message but keep the `sanitizeNumberString` call?